audioswitch/src/main/java/com/twilio/audioswitch/selection/AudioDeviceSelector.kt

```diff
@@ -27,6 +28,13 @@ private const val TAG = "AudioDeviceSelector"
  */
 class AudioDeviceSelector {
 
+    companion object {
+        /**
+         * The version of the AudioSwitch library.
+         */
+        const val VERSION = BuildConfig.VERSION_NAME
```

Comment:
I think we agreed on logging this at instantiation time right?

Reply:
I didn't see that in the AC. I think we discussed adding a prefix to log messages that included the version.

Reply:
Oh okay yep that works too. I'll include that in the logging story.